app/assets/stylesheets/email.css

```diff
@@ -0,0 +1,1198 @@
+table.header {
```

Comment:
remove this? or move any new styles to the view partial in `app/views/mail_shared` since that is what we are including for CSS Now